### ISSUE TITLE
job_pause causes AttributeError 'NoneType' object has no attribute 'strftime'

### ISSUE THREAD

**Issue by constantinoantunes** [jarekwg/django-apscheduler#41]
Using django-apscheduler==0.2.13:

- Add a job
- Pause the job
- Open /admin/django_apscheduler/djangojob/

The following error is presented:

> AttributeError at /admin/django_apscheduler/djangojob/
> 'NoneType' object has no attribute 'strftime'

It happens in: lib\site-packages\django_apscheduler\admin.py in next_run_time_sec, line 35

This method:
```
    def next_run_time_sec(self, obj):
        return obj.next_run_time.strftime("%Y-%m-%d %H:%M:%S")
```

**Comment by jcass77** [comment]
The ``next_run_time_sec`` implementation is also problematic for a different reason: if timezone support is activated via [USE_TZ](https://docs.djangoproject.com/en/dev/ref/settings/#use-tz), then the string output by ``next_run_time_sec`` will show UTC time, while the rest of django-admin shows the local time.

The way that django-apscheduler shows timezone-aware datetimes can be quite confusing as a result.

**Comment by D4VIDB2** [comment]
Can confirm that in django-apscheduler==0.3.0 the issue is still there.
Here is a workaround: #68

**Comment by jcass77** [comment]
Fixed via #68.
